consensus/broadcast/commonMessenger_test.go

```diff
@@ -7,10 +7,24 @@ import (
 	"github.com/ElrondNetwork/elrond-go/consensus/broadcast"
 	"github.com/ElrondNetwork/elrond-go/consensus/mock"
 	"github.com/ElrondNetwork/elrond-go/crypto"
+	"github.com/ElrondNetwork/elrond-go/data/block"
 	"github.com/pkg/errors"
 	"github.com/stretchr/testify/assert"
 )
 
+var (
+	testBlockBody = block.Body{
```

Comment:
done the funcrtion, but I don't see the connection/need with t.Parallel

Reply:
t.parallel just in case the tests can be run in parallel to speed up.
with the global testBlockBody it wouldn't have worked to run the tests in parallel due to race conditions on it.

Reply:
so are we good now?